plenum/test/test_node.py

```diff
@@ -367,7 +366,7 @@ def __init__(self, *args, **kwargs):
         self.ClientStackClass = clientStackClass
 
         Node.__init__(self, *args, **kwargs)
-        self.view_changer = TestViewChanger(self)
+        self.view_changer = TestViewChanger(ViewChangerNodeDataProvider(self))
```

Comment:
I highly recommend using here `create_view_changer(self, TestViewChanger)`

Reply:
Done. Thank you.